commit e56ffb0914d5498f4017b0046524009d2b617876
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Thu Sep 15 00:39:43 2016 +0200

    Make KF5Baloo optional
    
    Regardless of the current state of Baloo, it is not very deeply tied
    into Plasma. Usage in plasma-desktop comes down to providing the
    file search kcm. This makes it possible to prepare systems without a
    use for indexing.
    
    https://mail.kde.org/pipermail/kde-frameworks-devel/2016-September/037734.html
    
    REVIEW: 128957

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8c99352..b229dd5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,14 +67,10 @@ set_package_properties(PackageKitQt5 PROPERTIES DESCRIPTION "Software Manager in
                        PURPOSE "Provides package management integration to the application launcher."
                       )
 
-# Baloo is currently part of the KF5 namespace, but it is not
-# a proper framework. Hence the strange versioning.
-# This will be fixed with Plasma 5.2, as Baloo should have
-# become a framework by then
 find_package(KF5Baloo "5.24")
 set_package_properties(KF5Baloo PROPERTIES DESCRIPTION "File Searching"
-                       TYPE REQUIRED
-                       PURPOSE "Needed to build to File Search KCM"
+                       TYPE RECOMMENDED
+                       PURPOSE "Needed to build the File Search KCM"
                       )
 
 find_package(Fontconfig)
diff --git a/kcms/CMakeLists.txt b/kcms/CMakeLists.txt
index 53ca49a..3220bd3 100644
--- a/kcms/CMakeLists.txt
+++ b/kcms/CMakeLists.txt
@@ -55,7 +55,9 @@ add_subdirectory(useraccount/pics)
 
 
 add_subdirectory(workspaceoptions)
-add_subdirectory(baloo)
+if (KF5Baloo_FOUND)
+   add_subdirectory(baloo)
+endif()
 add_subdirectory(solid_actions)
 add_subdirectory(cursortheme)
 
